package bioinformatician;

//import basic_operations.*;
//import java.util.Scanner;

public class Bioinformatician_actions {
	public static void print_actions(){
		System.out.println("Welcome Bioinformatician!\nType the number of the action with which you want to procceed:");
		System.out.println("1.Display the alignment\n2.Display the SNiP alignment");
		System.out.println("3.Search for sequence in the standard alignment\n4.Search in the SNiP alignment\n5.Search for a genome title in the standard alignment");
		System.out.println("6.Search for a genome title in the SNiP alignment\n7.Display a genome in the standard alignment");
		System.out.println("8.Display a genome in the SNiP alignment\n9.Remove a Genome from the standard alignment.");
		System.out.println("10.Remove a Genome from the SNiP alignment.\n11.Add a Genome to the standard alignment.\n12.Add a Genome to the SNiP alignment.");
		System.out.println("13.Replace a genome in the standard alignment.\n14.Replace a genome in the SNiP alignment.");
		System.out.println("15.Replace a sequence of base pairs in the standard alignment");
		System.out.println("16.Replace a sequence of base pairs in the SNiP alignment");
		System.out.println("17.Replace a sequence of base pairs in one genome in the standard alignment");
		System.out.println("18.Replace a sequence of base pairs in one genome in the SNiP alignment");
		System.out.println("19.Change reference Genome in standard alignment.");
		System.out.println("20.Change reference Genome in SNiP alignment.");
		System.out.println("21.Find score of standard alignment.");
		System.out.println("22.Find score of SNiP alignment.");
		System.out.println("23.Save the changes you made to the alignment");
		System.out.println("24.Retrieve personal alignment");
	}
}
	